commit 8194fd1d715243daf9b15b52270eeba6c1a8db74
Author: Pascal van Leeuwen <pascalvanl@gmail.com>
Date:   Fri Sep 13 17:20:38 2019 +0200

    crypto: testmgr - Added testvectors for the hmac(sm3) ahash
    
    Added testvectors for the hmac(sm3) ahash authentication algorithm
    
    changes since v1 & v2:
    -nothing
    
    Signed-off-by: Pascal van Leeuwen <pvanleeuwen@verimatrix.com>
    Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>

diff --git a/crypto/testmgr.c b/crypto/testmgr.c
index c39e39e55dc2..40d94ad706d4 100644
--- a/crypto/testmgr.c
+++ b/crypto/testmgr.c
@@ -4654,6 +4654,12 @@ static const struct alg_test_desc alg_test_descs[] = {
 		.suite = {
 			.hash = __VECS(hmac_sha512_tv_template)
 		}
+	}, {
+		.alg = "hmac(sm3)",
+		.test = alg_test_hash,
+		.suite = {
+			.hash = __VECS(hmac_sm3_tv_template)
+		}
 	}, {
 		.alg = "hmac(streebog256)",
 		.test = alg_test_hash,